styles/components/_tabbedModal.scss

```diff
@@ -30,6 +30,10 @@
     }
   }
 
+  .tabContent {
```

Comment:
There may be a better way to do this, I'm still not 100% sure why the image blows things up on save.

The parent modal has a fixed width, I think that makes it ok for now to have fixed width columns in it. We could also do percentages.